app/src/main/java/de/tum/in/tumcampusapp/models/tumcabe/BarrierfreeContact.java

```diff
@@ -0,0 +1,71 @@
+package de.tum.in.tumcampusapp.models.tumcabe;
+
+import de.tum.in.tumcampusapp.adapters.SimpleStickyListHeadersAdapter;
+
+/**
+ * The model used to display contact infromation in barrier free page
+ */
+public class BarrierfreeContact implements SimpleStickyListHeadersAdapter.SimpleStickyListItem{
+    private String name;
+    private String telephone;
+    private String email;
+    private String faculty;
+    private String tumID = "";
+
+    public BarrierfreeContact(String name, String phone, String email, String faculty, String tumonlineID){
+        this.name = name;
+        this.telephone = phone;
+        this.email = email;
+        this.faculty = faculty;
+        this.tumID = tumonlineID;
+    }
+
+    public boolean ifValid(){
```

Comment:
Usually our boolean methods start with `is`.
E.g. `isValid()` and `isTumIDValid()`